doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
 
-The networking within a cloud infrastructure, fabric or otherwise, is an area where there is significant variability across implementations. Leaf-Spine topology is well established, however, after topology there are countless decisions an Operator needs to make. Differences arise from many aspects, for example, is the solution layer-2 or layer-3; is the routing static or dynamic; what mechanism is used for encapsulation; what mechanism is used for isolation; does it support SR-IOV; does it support DPDK; does it employ SmartNICs; does it employ distributed control or a centralized control driving a programmable fabric; and the list continues. The multitude of permutations enable cloud infrastructure architects (Operators and Suppliers) to design (or procure) a fabric/networking solution that's optimized for their needs, whether their needs are minimal, very extensive or somewhere in between.
+<a name="5.1.1.1"></a>
+#### 5.1.1.1 Identified Networking Gaps
+To start the community discussion around potential gaps in the body of existing standards around the Cloud Infrastructure (NFV Infrastracture) networking, we looked into the ETSI NFV Architectural Framework. We have identified a number of areas that, in our view, need to be addressed (see below for the initial list). As usual, in our further efforts, the community will need to decide where and how these items should be addressed.
 
-For CNTT, a strategy is needed that affords Operators the performance, flexibility, availability, maintainability and scalability their business requires, yet doesn't require OPNFV to design, manage and test prohibitive numbers of networking solutions.
- 
-**Some points for CNTT to consider when contemplating recommendations:**
-   > * Despite large variances in implementation, the spectrum of networking capabilities ultimately delivered to Workloads is comparatively narrow
-   > * Standard CNTT methodology (i.e. normalize interfaces, APIs, capabilities and behaviors at the reference points) applies well to networking, helping mitigate the need to be overly prescriptive about implementation  
-   > * CNTT/OPNFV MUST provide a functional networking solution for the RI, and in support of RC  
-   > * CNTT/OPNFV potentially does NOT have to provide a production networking solution for RI  
-   > * CNTT/OPNFV is NOT planning to provide a production compute solution for RI
+•	ETSI NFV does not have a separation between HW Infrastructure Management and SW Virtualisation Management and by that the cardinality of having multiple CaaS and IaaS layers on top of a shared HW Infrastructure Layer cannot be expressed
 
+•	ETSI NFV lacks a description of the reference points in between the SW Virtualisation Layers and the HW Infrastructure Layer denoted as Vl-Ha and by that cannot express Packet Flows, Control/Status Interfaces and Management Interfaces between them
```

Comment:
Suggest more neutral wording than "ETSI lacks". ETSI ISG NFV on purpose did not describe existing state-of-the-art interfaces, but leaves this to implementation.
In our CNTT work flow that means interfaces between constituents of the cloud infrastructure may be different for different RIs.

Reply:
@ulikleber I agree on that a more neutral wording would be better.
I think it is fine that ETSI NFV do not specify too deep in some areas.
CNTT however has the target of making it simpler for the operators to choose VNFs/CNFs as well as how they like to build or procure their Infrastructure Layers. In that quest CNTT must be more prescriptive.
The interfaces in between the constituents become important mostly to support the choise of Infrastructure support and selection of VNFs/CNFs, not the RI, since the RI is a tool for the RC to show Conformance level or fulfillment. RI is, as far as I have understood CNTT, not a productified blueprint for an operator production environment or an offering from a vendor.
Having said that, I agree that interfaces between constituents in different RI specifically stemming from different RA and they could theoretically also differ between RIs based on the same RA, but I feel it is unlikely that there will exist many RI per RA due to its task to support the RC for Badging purposes.

Reply:
@ulikleber as per my previous comments about "gaps".  But I take your point and will explain that ETSI doe snot go to this level in this areas.  That forms these "gaps" which we have to fill. As everywhere, you have higher level standards and lower level standards supporting them. I agree with @TFredberg in explaining why we have to be more prescriptive so I will not repeat it here.

Reply:
"...and by that cannot express **_the_** Packet Flows..." - Add "the"